pkg/features/kube_features.go

```diff
@@ -108,14 +108,6 @@ const (
 	// Allow the usage of options to fine-tune the cpumanager policies.
 	CPUManagerPolicyOptions featuregate.Feature = "CPUManagerPolicyOptions"
 
-	// owner: @leakingtapan
-	// alpha: v1.14
-	// beta: v1.17
-	// GA: v1.25
-	//
-	// Enables the AWS EBS in-tree driver to AWS EBS CSI Driver migration feature.
-	CSIMigrationAWS featuregate.Feature = "CSIMigrationAWS"
```

Comment:
weirdly, this looks like it made tests like https://testgrid.k8s.io/sig-scalability-benchmarks#scheduler-perf start failing ... but I don't see this gate set in the job config ... any ideas?

cc @alculquicondor for sig-scheduling

Reply:
#116359